open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def get_Histogram_of_AgeFill_segmented_by_Survived(df, column_number):
</pre><p> </p><p> </p><p>def get_age_by_survived(df, survived):</p><p> </p><p> </p><p>And please add documentation and questions for this function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>done</p>